core/api/src/app/authentication/login.ts

```diff
@@ -104,6 +106,12 @@ export const loginWithPhoneToken = async ({
     })
     if (kratosResult instanceof Error) return kratosResult
 
+    const account = await createAccountFromRegistrationPayload({
+      secret: KRATOS_CALLBACK_API_KEY,
```

Comment:
This is the only place we're applying the callback api key now, and it's pretty much redundant here. Should I remove?

Reply:
yes `secret` can be removed completely, together with `KRATOS_CALLBACK_API_KEY` (also in charts/deployment)

Reply:
I think you could call `createAccountWithPhoneIdentifier` directly here

Reply:
ie: 

```
  const account = await createAccountWithPhoneIdentifier({
    newAccountInfo: { phone, kratosUserId: userId },
    config: getDefaultAccountsConfig(),
    phoneMetadata,
  })
```

Reply:
Resolved https://github.com/GaloyMoney/galoy/pull/3925/commits/ce81cf72bbd549e0434336b0642e9907a0164780